src/spikeinterface/core/waveform_extractor.py

```diff
@@ -517,15 +524,19 @@ def is_extension(self, extension_name) -> bool:
         if self.folder is None:
             return extension_name in self._loaded_extensions
         else:
```

Comment:
You don't need this else either. This will allow you to get rid of the nesting.

Reply:
Done